assignment.js

```diff
@@ -111,6 +121,36 @@ Use the following test cases to confirm your program meets the success criteria
 
 console.log('Problem 3:')
 
+const playerAChoice = "paper"
+const playerBChoice = "scissors"
+
+if(playerAChoice === playerBChoice){
+  console.log("It's a tie!")
+  }else if(playerAChoice === "rock"){
+    if(playerBChoice === 'scissors'){
+      console.log("playerA wins")
+      }else{
+        console.log("playerB wins") //
+        }
+  }else if(playerAChoice === "scissors"){
+    if(playerBChoice === "paper"){
+      console.log("playerA wins")
+      }else{
+        console.log("playerB wins") //scissors +rocks
+        }
+    }else if(playerAChoice === "paper"){
+      if(playerBChoice === 'rock'){
+        console.log("playerA wins")
+        }else{
+          console.log("playerB wins") //
+          }
+      }
+
```

Comment:
@nmikhol  Good work here utilizing the loops and if statements to complete this task. I would suggest adding one more conditional that would return an error if the player's choice is wrong. Take a look at this approach [here](https://github.com/jsr20190826/hw-01-js-basics-control-flow-loops-solution/blob/a810fe3f581584e5015519a3508b078d38c4c8c9/assignment.js#L150)

Reply:
Nice indentation and a good job making that change